memory-viz/src/tests/__snapshots__/snapshot.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>SVG Snapshot Viewer</title>
+    </head>
+    <body>
+        <label>
+            Load a Jest Snapshot Output
+            <input
```

Comment:
use the `accept` attribute to look for `.snap` files (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept)